CONTRIBUTING.md

```diff
@@ -0,0 +1,11 @@
+# Contributing
+
+Contributions, issues and feature requests are very welcome. If you are using this app and fixed a bug or improve it, please consider submitting a PR!
```

Comment:
```suggestion
Contributions, issues and feature requests are very welcome. If you are using this app and improved it or fixed a bug, please consider submitting a PR!
```

- grammatical error "improve" -> "improved"
- I suggest inverting "improved it" and "fixed a bug" so that there is no grammatical ambiguity as to what "improve" refers to.